From 83219d864c2ddbc0cbd79a021f2527e0d0c38607 Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Fri, 19 Oct 2018 12:16:33 +0200
Subject: [PATCH] Merge android-9.0.0_r12

commit 4857e631f64cc981eaf353dca7b609f8ecbe94e2
Merge: 5c3a89ea 10cde684
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Sun Jul 15 03:09:25 2018 +0000

    Snap for 4894342 from 10cde684808f7b6c68dffd0f2f48cce8b50ec0ca to pi-dr1-release

    Change-Id: If780a637b6615067e40096d939105886e651a7ac

commit 10cde684808f7b6c68dffd0f2f48cce8b50ec0ca
Merge: ce9f204a 9c2867c8
Author: Jeff Sharkey <jsharkey@android.com>
Date:   Thu Jul 12 20:57:48 2018 -0700

    [automerger] Remove "public" download feature. am: 35e123117b am: 52b9a670b8 am: 7f0b5c44c8 am: 3439b74e4a am: b2db30a21e am: 84cefc516e am: 32a92d0a73
    am: 9c2867c8a1

    Change-Id: I93ec2760b7db41dcb551bcdf16c6d38df078ab52

commit 9c2867c8a12ad9bc04a1d8bdd4fc3879f3ffc178
Merge: b627002e 32a92d0a
Author: Jeff Sharkey <jsharkey@android.com>
Date:   Thu Jul 12 19:05:17 2018 -0700

    [automerger] Remove "public" download feature. am: 35e123117b am: 52b9a670b8 am: 7f0b5c44c8 am: 3439b74e4a am: b2db30a21e am: 84cefc516e
    am: 32a92d0a73

    Change-Id: Ib38b3c09bfaf7e159712b09b6c8f1b82ed33833d

commit 32a92d0a737d42af54f44b9f1128fa7ea676f13a
Merge: b2de5829 84cefc51
Author: Jeff Sharkey <jsharkey@android.com>
Date:   Thu Jul 12 17:30:20 2018 -0700

    [automerger] Remove "public" download feature. am: 35e123117b am: 52b9a670b8 am: 7f0b5c44c8 am: 3439b74e4a am: b2db30a21e
    am: 84cefc516e

    Change-Id: Iabab984d78bad2ca6b5904e31482044e4627d02a

commit ce9f204ac493f000cd3020e195fd5038d0cec1e2
Author: Jeff Sharkey <jsharkey@android.com>
Date:   Mon Jul 9 12:16:26 2018 -0600

    Remove "public" download feature.

    It was never a supported API, and has been reported as causing
    security issues, so remove it.

    Bug: 111084083
    Test: builds
    Change-Id: I26345b192ffd55216bb8c8fdb82cb5869d68d3db

commit 84cefc516ed8691b11670b1cfd80216969cfbfef
Merge: d08c19e2 b2db30a2
Author: Android Build Merger (Role) <noreply-android-build-merger@google.com>
Date:   Mon Jul 9 18:18:03 2018 +0000

    [automerger] Remove "public" download feature. am: 35e123117b am: 52b9a670b8 am: 7f0b5c44c8 am: 3439b74e4a am: b2db30a21e

    Change-Id: I4e2fda8c302ddfaa0edeb226029c94ff3771f89f

commit b2db30a21eaac0ebc033bc5a824b146633bedda0
Merge: 3a1bec71 3439b74e
Author: Android Build Merger (Role) <noreply-android-build-merger@google.com>
Date:   Mon Jul 9 18:18:02 2018 +0000

    [automerger] Remove "public" download feature. am: 35e123117b am: 52b9a670b8 am: 7f0b5c44c8 am: 3439b74e4a

    Change-Id: I1509a4dc66f4d57701d54b7743de3bd9915258c7

commit 3439b74e4af9c7303310545736578d1fcf518412
Merge: 09fb61e7 7f0b5c44
Author: Android Build Merger (Role) <noreply-android-build-merger@google.com>
Date:   Mon Jul 9 18:18:01 2018 +0000

    [automerger] Remove "public" download feature. am: 35e123117b am: 52b9a670b8 am: 7f0b5c44c8

    Change-Id: I7b24ea92c1227e4581e72b551e3ebac4b4bd3528

commit 7f0b5c44c87b156d050d8183603ed2d828e0a874
Merge: 79f65e6e 52b9a670
Author: Android Build Merger (Role) <noreply-android-build-merger@google.com>
Date:   Mon Jul 9 18:17:59 2018 +0000

    [automerger] Remove "public" download feature. am: 35e123117b am: 52b9a670b8

    Change-Id: I55c96981db12008e619333f67ea8033e79ca5bd2

commit 52b9a670b80f5843316c4e076a40abd178e28b9c
Merge: bf2ee732 35e12311
Author: Android Build Merger (Role) <noreply-android-build-merger@google.com>
Date:   Mon Jul 9 18:17:58 2018 +0000

    [automerger] Remove "public" download feature. am: 35e123117b

    Change-Id: I0347a0a15e99a224df13b756dc3d27de23835cae

commit 35e123117be9ec5d61dbaea60f6eac06c0e80dc4
Author: Jeff Sharkey <jsharkey@android.com>
Date:   Mon Jul 9 12:16:26 2018 -0600

    Remove "public" download feature.

    It was never a supported API, and has been reported as causing
    security issues, so remove it.

    Bug: 111084083
    Test: builds
    Change-Id: I26345b192ffd55216bb8c8fdb82cb5869d68d3db

commit 5c3a89eaa3dbf7b4a2b06267f1c69a64e48f0eed
Merge: 50dee288 aeda68c6
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Sun Jun 24 03:06:26 2018 +0000

    Snap for 4859140 from aeda68c64c46fd3ea28bbaa78d5c4f89b49eadd9 to pi-dr1-release

    Change-Id: Ib5ea34e85a05092c096193d8eb81add7a42eea4c

commit aeda68c64c46fd3ea28bbaa78d5c4f89b49eadd9
Author: Bill Yi <byi@google.com>
Date:   Thu Jun 21 09:04:59 2018 -0700

    Import translations. DO NOT MERGE

    Auto-generated-cl: translation import

    Bug: 64712476
    Change-Id: Ibe041341d032a0eb9b0f31b74db37c2e3de38df9

Change-Id: I2008833997bbf7077365dee64b7a3a6c1075e282
---
 res/values-fr/strings.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/res/values-fr/strings.xml b/res/values-fr/strings.xml
index ed341cc..83489be 100644
--- a/res/values-fr/strings.xml
+++ b/res/values-fr/strings.xml
@@ -16,7 +16,7 @@
 
 <resources xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-    <string name="app_label" msgid="3658948994665187911">"Gest. de tél."</string>
+    <string name="app_label" msgid="3658948994665187911">"Gestionnaire de téléchargement"</string>
     <string name="storage_description" msgid="7982444311558023664">"Téléchargements"</string>
     <string name="permlab_downloadManager" msgid="7779544811202855500">"Accéder au gestionnaire de téléchargement."</string>
     <string name="permdesc_downloadManager" msgid="4237406545998908947">"Permet à l\'application d\'accéder au gestionnaire de téléchargement et de l\'utiliser pour télécharger des fichiers. Les applications malveillantes peuvent utiliser cette fonctionnalité pour perturber les téléchargements et accéder à des informations confidentielles."</string>
-- 
2.17.1

